package/MDAnalysis/topology/TXYZParser.py

```diff
@@ -120,26 +125,29 @@ def parse(self, **kwargs):
                     if i < other_atom:
                         bonds.append((i, other_atom))
 
-        # Guessing time
-        masses = guessers.guess_masses(names)
+        for n in names:
+            if n.capitalize() in SYMB2Z:
+                validated_elements.append(n.capitalize())
+            else:
+                validated_elements.append('')
```

Comment:
The changelog mentions adding an element attribute for this - but it seemed to already be available on line 136. Is this `validate_elements` being used anywhere?

Reply:
that looks weird tbh cause I already added elements it in a separate pr.... I messed this up